api/resources/views/admin/roles/edit.blade.php

```diff
@@ -0,0 +1,44 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="card">
+            <div class="card-body">
+                <div class="form-group row">
+                    <label class="col-md-4 col-form-label text-md-right">Current role</label>
+
+                    <div class="col-md-6">
+                        <label class="col-form-label">Admin</label>
+                    </div>
+                </div>
+                <form method="POST" class="align-content-center">
+                    @csrf
+                    <div class="form-group row">
+                        <label for="role" class="col-md-4 col-form-label text-md-right">New role</label>
+
+                        <div class="col-md-6">
+                            <select name="role">
```

Comment:
Hard coded roles. I understand that this is only prototype but don't forget to change it in further work.

Reply:
Will do, placeholders because roles are not implemented yet.